.github/workflows/build.yml

```
@@ -0,0 +1,126 @@
+      - name: Download macOS DMG
+        uses: actions/download-artifact@v4
+        with:
+          name: barcode-xpress-macos-dmg
+
+      - name: Download Windows executable
+        uses: actions/download-artifact@v4
+        with:
+          name: barcode-xpress-windows
+
+      - name: Create Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.8)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)